x-pack/plugin/esql/qa/testFixtures/src/main/resources/enrich.csv-spec

```diff
@@ -31,6 +31,105 @@ FROM sample_data
 median_duration:double | env:keyword
 ;
 
+withQualifierConflict
+required_capability: enrich_load
+required_capability: enrich_with_qualifier
+ROW client_ip = "172.21.0.5", env = 1
+| ENRICH clientip_policy ON client_ip AS c
```

Comment:
If c is an alias of clients_policy, is it better to keep it closer to it? For example, `ENRICH clientip_policy as c ON client_ip` or `ENRICH clientip_policy c ON client_ip`?

Reply:
Yeah, this works, too, and is closer to SQL!

One reason for having the `AS c` at the end is that it takes the place of `WITH` for renaming fields; and `ENRICH policy AS c` sounds like we're optionally renaming the policy, which suggests that `ENRICH policy | KEEP policy.field` would work, but that doesn't.

Anyway, I don't feel strongly about the exact syntax and think all of that works!